doc/ref_model/chapters/chapter08.rst

```diff
@@ -213,118 +305,231 @@ Requirements, Reference Architecture & Industry Standards Intersect
-
-======================= ================= ===== ========= ======= ========== ============= ========= ========== ================ ================= ================== =================== =======
-\                       Platform Services                                                             Storage                                       Network Services
-======================= ================= ===== ========= ======= ========== ============= ========= ========== ================ ================= ================== =================== =======
-\                       Identity          Image Placement Compute Networking Message Queue DB Server  Ephemeral Persistent Block Persistent Object  Management        Underlay (Provider) Overlay
-Control Nodes           ✅                ✅    ✅        ✅      ✅          ✅           ✅                   ✅                                  ✅                ✅                  ✅
-Workload Nodes(Compute)                                   ✅      ✅                                  ✅        ✅               ✅                 ✅                ✅                  ✅
-Storage Nodes                                                                                                   ✅               ✅                 ✅                ✅                  ✅
-======================= ================= ===== ========= ======= ========== ============= ========= ========== ================ ================= ================== =================== =======
+This section characterises the hardware capabilities for different edge deployments and the Platform services that run
+on the infrastructure. Please note, that the Platform services are containerised to save resources, and benefit from
+intrinsic availability and auto-scaling capabilities.
+
++-----------+--------------------------------------------------------+-------------------------+-----------------------+
+|           | Platform Services                                      | Storage                 | Network Services      |
+|           +-------+-------+-------+-------+-------+-------+--------+--------+--------+-------+-------+-------+-------+
+|           | Iden- | Image | Plac- | Comp- | Netw- | Mess- | DB     | Ephem- | Persi- | Pers- | Mana- | Unde- | Over- |
+|           | tity  |       | ement | ute   | orki- | age   | Server | eral   | stent  | iste- | geme- | rlay  | lay   |
+|           |       |       |       |       | ng    | Queue |        |        | Block  | nt    | nt    | (Pro- |       |
+|           |       |       |       |       |       |       |        |        |        | Obje- |       | vid-  |       |
+|           |       |       |       |       |       |       |        |        |        | ct    |       | er)   |       |
++===========+=======+=======+=======+=======+=======+=======+========+========+========+=======+=======+=======+=======+
+| Control   | ✅    | ✅    | ✅    | ✅    | ✅    | ✅    | ✅     |        | ✅     |       | ✅    | ✅    | ✅    |
```

Comment:
looks like the padding is off by one.

Reply:
It is off only due to the tickmark characters (✅). 
It renders nicely: 
![image](https://user-images.githubusercontent.com/11458775/168245441-9b5a5f4f-5395-4b3c-b182-6711b91f36a2.png)

Reply:
ah, ok. that explains it. Thanks